src/engine/controls/cuecontrol.cpp

```diff
@@ -670,6 +802,69 @@ void CueControl::hotcueGotoAndPlay(HotcueControl* pControl, double v) {
     }
 }
 
+void CueControl::hotcueReloop(HotcueControl* pControl, double v) {
+    if (!v)
+        return;
+
+    QMutexLocker lock(&m_mutex);
```

Comment:
I don't really know tbh. Since that locking code also existed in similar functions (e.g. `hotcueGoto` [(see here)](https://github.com/mixxxdj/mixxx/blob/45a6fb57c6cf5b30b459144391833984f39a8df2/src/engine/controls/cuecontrol.cpp#L734-L742)), I just thought it's probably needed and added it, too.

Reply:
The cues can be altered, but the lock also guards `hotcueClear`, including the `detachCue` call . I'd rather not risk breaking it in this PR and leave it in for now until someone cleans up the locking logic in a separate PR.

Reply:
I added a comment though.